client/components/CheckoutCalendar/CheckoutCalendar.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import axios from 'axios';
+import { numberOfGuests, totalReviewCount, averageReviewRatings } from '../../../sampleData/sampleData';
+
 
 class CheckoutCalendar extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      allowedGuests: numberOfGuests.numberOfGuests,
+      totalReviews: totalReviewCount,
+      averageRating: averageReviewRatings.averageRating,
+      pricePerNight: '',
+    };
   }
 
   componentDidMount() {
-    // axios.get('/checkoutInformation/10')
-    //   .then(checkoutInfo => {})
+    console.log('inside axios')
+    axios.get('/checkoutInformation/10')
+      .then(checkoutInfo => {
+        this.setState({ pricePerNight: checkoutInfo.data.priceForDate })
+      });
   }
 
   render() {
-    return(
+    const { allowedGuests, totalReviews, averageRating, pricePerNight } = this.state;
```

Comment:
i really like the use of destructuring here. i need to do this more/do this at all.